qemu/tests/hv_enforce_cpuid_smoke.py

```diff
@@ -0,0 +1,34 @@
+from virttest import error_context
+from provider import qemu_img_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Add 'hv-enforce-cpuid' option to see whether guest use features we didn't advertize to it.
+    1. Boot the guest with cpu_model_flags.
+    2. login
+    param test: the test object
+    param params: the test params
+    param env: the test env object
+    """
+
+    res = []
+    cpu_model_flags_list = params.objects("cpu_model_flags_list")
+    for cpu_model_flags in cpu_model_flags_list:
+        try:
+            params["cpu_model_flags"] = cpu_model_flags
```

Comment:
better add a message to make others know the test step clearly. (it will be more clear in log)
eg:
error_context.context("Start the guest with %s"， cpu_model_flags, test.log.info)